assets/sass/components/global/_googlesitekit-stepper.scss

```diff
@@ -91,16 +79,54 @@
 		}
 	}
 
-	.googlesitekit-stepper__step-title--upcoming {
-		color: $c-surfaces-on-surface-variant;
+	.googlesitekit-stepper__step-content-container {
+		display: grid;
+		grid-template-rows: 0fr;
+		transition: grid-template-rows $t-default ease-in-out;
```

Comment:
Very nice! I totally didn't spot the possibility of transitioning `grid-template-rows` while drafting the IB. Makes for a pure CSS approach, which is great to see.